finary_uapi/user_real_estates.py

```diff
@@ -89,7 +92,13 @@ def add_user_real_estates(
     headers = {}
     headers["Content-Length"] = str(len(data_json))
     headers["Content-Type"] = "application/json"
-    x = session.post(url, data=data_json, headers=headers)
+    if currency_code == get_display_currency_code(session):
```

Comment:
I'm not against this type of thing but I would prefer to have a "pure" API function with a mandatory `currency_code` and an utils function without it and doing the check in display settings and calling the pure API one. I believe it makes the maintenance easier.

Reply:
Isn’t what I did?
I actually added currency code into add real estate function.
However as of today currency code is not mandatory in this case as using session display currency by default.
Or maybe I’m misunderstanding what you said 😅